### ISSUE TITLE
Add Title attribute for helpful text for "Check Integrity" button

### ISSUE THREAD

**Issue by jamesmaa** [yomidevs/yomitan#1203]
Not many people know what this is. Let's add helpful text here for users so they know what's up. Even I don't really know what "Check Integrity" does.

<img width="649" alt="image" src="https://github.com/themoeway/yomitan/assets/2382537/1fcc43e8-80ee-427d-b97a-6a634d703c27">

You just need to add a `title` to the button html element

https://stackoverflow.com/a/52934531

**Comment by khaitruong922** [comment]
Let's try adding title for this button first before adding title to other buttons. I don't think we need title for buttons where functionality is obvious.

The Check Integrity button will add a button before the Priority field which will opens a popup with the metadata of dictionaries.

<img width="590" alt="Screenshot 2024-07-12 at 11 59 29" src="https://github.com/user-attachments/assets/9ee26fce-3944-4fbe-9acc-e8b6a8e18a2e">

I would suggest something like "Display metadata of dictionaries". What do you guys think?

**Comment by jamesmaa** [comment]
> The Check Integrity button will add a button before the Priority field which will opens a popup with the metadata of dictionaries.

I'm not following what you're saying. Are you proposing a different kind of change?

**Comment by khaitruong922** [comment]
No, I'm just describing what the Check Integrity button does. It creates the tick button before the priority field where you can view the metadata of the dictionaries.

<img width="603" alt="Screenshot 2024-07-17 at 15 28 48" src="https://github.com/user-attachments/assets/507c0c05-0d51-4a87-9f62-85fe438ff932">

**Comment by harukanon** [comment]
I think the functionality should be placed in the more button to make it less confusing.

**Comment by ganqqwerty** [comment]
I'm one of the confused users here. Why not displaying this tick button all the time? Is Check Integrity button even needed? In which cases is it used?

**Comment by h1divp** [comment]
Hi there, does this issue still need to be fixed?

**Comment by Kuuuube** [comment]
Closing as this probably doesn't matter anymore. This setting has been moved to debug settings and almost no users will see it anyways. @h1divp if you would like to fix it, feel free. PRs for this will still be accepted.
